CourseGrab/Home/HomeTableViewCell.swift

```diff
@@ -8,7 +8,152 @@
 
 import Foundation
 import UIKit
+import SnapKit
 
 class HomeTableViewCell: UITableViewCell {
     
+    var section: Section? {
+        didSet {
+            guard let section = section else { return }
+            
+            courseLabel.text = String(section.courseNum)
+            enrollButton.isHidden = section.status != .open
+            sectionLabel.text = section.section
+            statusBadge.image = section.status.icon
+            titleLabel.text = section.title
+
+            if section.status == .open {
+                removeButton.snp.makeConstraints { make in
+                    make.top.equalTo(sectionLabel.snp.bottom).offset(12)
+                    make.leading.equalTo(16)
+                    make.height.equalTo(24)
+                    make.trailing.equalTo(container.snp.centerX).inset(9)
+                    make.bottom.equalTo(container).inset(16)
+                }
+            } else {
+                removeButton.snp.remakeConstraints { make in
+                    make.top.equalTo(sectionLabel.snp.bottom).offset(12)
+                    make.leading.equalTo(16)
+                    make.height.equalTo(24)
+                    make.trailing.equalTo(container).inset(16)
+                    make.bottom.equalTo(container).inset(16)
+                }
+            }
+        }
+    }
```

Comment:
instead of having this big `didSet` at the beginning of the file, can we make a `func configure(for section: Section)` function and move this logic into there. This means that you would just do `cell.configure(for: section)` in the `cellForRowAt` function

Reply:
This is how I did it in the SearchDetailTableVC, so it should remain consistent

Reply:
if this is the case, can we switch both instances to use this `configure` function?